lib/multiSig-tasks/update-v2-to-v3.ts

```diff
@@ -0,0 +1,180 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types";
+import {
+  getSignerAndSetDeploymentPath,
+  setLocalNodeDeploymentPath,
+  TransactionArguments,
+} from "../helpers/interfaceHelper";
+import {
+  ACCOUNT,
+  ACCOUNT_DESCRIPTION,
+  MULTISIG_UPDATE_V2_V3_DESCRIPTION,
+  OWNER_ADDRESS,
+  OWNER_ADDRESS_DESCRIPTION,
+} from "../helpers/staticVariables";
+import { taskLogger } from "../logger";
+import {
+  MULTISIG_ENCODE_PROPOSAL_PAYLOAD,
+  MULTISIG_SUBMIT_PROPOSAL,
+  MULTISIG_UPDATE_V2_V3,
+} from "../tasksNames";
+
+const ADDRESS_ZERO = "0x0000000000000000000000000000000000000000";
+
+task(MULTISIG_UPDATE_V2_V3, MULTISIG_UPDATE_V2_V3_DESCRIPTION)
+  .addParam(OWNER_ADDRESS, OWNER_ADDRESS_DESCRIPTION, undefined, types.string)
+  .addOptionalParam(ACCOUNT, ACCOUNT_DESCRIPTION, undefined, types.string)
+  .setAction(async (args: TransactionArguments, hre) => {
+    try {
+      taskLogger.setLogLevel("info");
+      taskLogger.info(`${MULTISIG_UPDATE_V2_V3} task...`);
+      await setLocalNodeDeploymentPath(hre);
+      const destinations: string[] = [];
+      const values: number[] = [];
+      const payloads: string[] = [];
+
+      const signer = await getSignerAndSetDeploymentPath(hre, args);
+
+      await generateContractUpdate(hre, "MultiSig", destinations, values, payloads);
+      await generateContractUpdate(hre, "Manager", destinations, values, payloads);
+      await generateContractUpdate(hre, "Account", destinations, values, payloads);
+      await generateContractUpdate(hre, "StakedCelo", destinations, values, payloads);
+      await generateContractUpdate(hre, "Vote", destinations, values, payloads);
+      await generateContractUpdate(hre, "GroupHealth", destinations, values, payloads);
+      await generateContractUpdate(hre, "SpecificGroupStrategy", destinations, values, payloads);
+      await generateContractUpdate(hre, "DefaultStrategy", destinations, values, payloads);
+      await generateContractUpdate(hre, "RebasedStakedCelo", destinations, values, payloads);
+
+      await generateMultiSigSetPauser(hre, "MultiSig", destinations, values, payloads);
+      await generateSetPauser(hre, "Manager", destinations, values, payloads);
+      await generateSetPauser(hre, "Account", destinations, values, payloads);
+      await generateSetPauser(hre, "StakedCelo", destinations, values, payloads);
+      await generateSetPauser(hre, "Vote", destinations, values, payloads);
+      await generateSetPauser(hre, "GroupHealth", destinations, values, payloads);
+      await generateSetPauser(hre, "SpecificGroupStrategy", destinations, values, payloads);
+      await generateSetPauser(hre, "DefaultStrategy", destinations, values, payloads);
+      await generateSetPauser(hre, "RebasedStakedCelo", destinations, values, payloads);
+
+      await generateSetMinCountOfActiveGroups(hre, destinations, values, payloads);
+      await generateMultiSigAddOwner(hre, args.ownerAddress!, destinations, values, payloads);
```

Comment:
Should we hardcode the owner so we cannot make mistake when running the command manually?

Reply:
will hardcode